tests/test_mdlp.py

```diff
@@ -32,165 +34,52 @@ def test_find_cut_no_cut():
     k = find_cut(y, 0, len(y))
     assert_equal(-1, k)
 
-def test_mdlp_iris():
-    from sklearn.datasets import load_iris
+def test_e2e():
+  X = np.array([
+    [0.1, 0.1],
+    [0.2, 1.1],
+    [0.3, 0.2],
+    [1.5, .9]
```

Comment:
You can also change 1.5 -> 0.4 correct?

Reply:
Indeed. That is true. I will change the data here to remove some small red herrings as to what is going on.